lib/src/resources/base_url.dart

```diff
@@ -0,0 +1,5 @@
+abstract class BaseUrl {
```

Comment:
If you want you can use direct const they shouldnt be in a class, unless you want them organized